### ISSUE TITLE
[Idea] Trophies more options

### ISSUE THREAD

**Issue by DracoBluePL** [WoltLab/WCF#5849]
1. The option to upload your own icon in SVG format (vector).
2. More customization options for visualizing trophies, such as borders, gradients, etc. To enhance their appearance and make them look nicer.

**Comment by IceQ1337** [comment]
What is stopping you from styling (individual) trophies using global CSS? Adding logic to handle all these things is, in my opinion, disproportionate to a bit of CSS and personal initiative. For example, adding a linear-gradient with different colors for specific trophies takes about 7 lines of SCSS.

**Comment by DracoBluePL** [comment]
Sorry, but that's what trophy functions and current customization are for, which can be easily expanded. I don't manage trophies in one place just to adjust their appearance in CSS code... If I wanted to customize everything like that, I wouldn't need a management panel. There are functions that can be improved. Currently, you can change the color of trophies, so why not add gradients? And perhaps borders? The purpose of these functions is to develop them, not to keep them stagnant for another decade.

**Comment by IceQ1337** [comment]
I disagree. In my opinion, this level of customization is exactly what styles and custom CSS are for.

**Comment by DracoBluePL** [comment]
You disagree because you know CSS, many forum owners don't. WoltLab wasn't created solely for programmers but also for laypeople, and if there's a trophy system, it can be expanded with new features rather than letting laypeople mess with code. You look at it from a different perspective.

**Comment by cadeyrn** [comment]
I support the first suggestion, but not the second. This level of customization can be done with CSS, as IceQ1337 said. If there are options for details such as borders, even for tiny features like trophies, this goes into far too much detail. If you start there, where do you stop? More styling options for user styles and avatars?

By the way, CSS has nothing to do with "programming" at all. And simple adjustments like adding a border can be done by everyone. It's not that you have to learn anything. I know what I am talking about. In my forum, we have users every day who customize their Firefox with CSS and have never had anything to do with CSS before.

**Comment by DracoBluePL** [comment]
And I stand by my opinion. Features like trophies, etc., should be improved over time rather than kept stagnant for another decade. Just look at what the software leading this market is doing and learn from them.

**Comment by cadeyrn** [comment]
> Features like trophies, etc., should be improved over time rather than kept stagnant for another decade

Yes. But this doesn't mean that _every_ suggestion is a good suggestion. Again, I support your first suggestion. And I am sure that there is potential for other improvements, too. But I do not think that it's a good investment to make complex style builders for every tiny feature of the software.
